.github/labeler.yml

```diff
@@ -64,3 +86,7 @@ Github:
       - .github/**/*
       - .github/*
       - README.md
+
+invalid:
+      - .stickler.yml
+      - .hound.yml
```

Comment:
LGTM! We are now dropping support for stickler (and possibly HoundCI) (as part of #55)